resources/js/requests/components/FilterMobile.vue

```diff
@@ -0,0 +1,217 @@
+<template>
+  <div class="row">
+    <div class="col-md-8 offset-md-2 col-lg-6 offset-lg-3">
+      <div>
+        <b-nav pills>
+          <div
+            class="dropdown"
+            left
+          >
+            <button
+              class="btn btn-secondary dropdown-toggle"
+              type="button"
+              id="dropdownMenuButton"
+              data-toggle="dropdown"
+              aria-haspopup="true"
+              aria-expanded="false"
+            >
+              <i class="fas fa-circle text-warning"></i>
+              {{ selectedOption }}
+              <i class="fas fa-caret-down"></i>
+            </button>
+            <div
+              class="dropdown-menu"
+              aria-labelledby="dropdownMenuButton"
+            >
+              <a
+                class="dropdown-item"
+                href="#"
+                @click="selectOption('In Progress', 'status')"
+                ><i class="fas fa-circle text-warning"></i> In Progress</a
+              >
+              <a
+                class="dropdown-item"
+                href="#"
+                @click="selectOption('Completed', 'status')"
+                ><i class="fas fa-circle text-primary"></i> Completed</a
+              >
+            </div>
+          </div>
+
+          <div
+            class="dropdown"
+            v-if="type === 'requests'"
+          >
+            <button
+              class="btn btn-secondary dropdown-toggle"
+              type="button"
+              id="dropdownMenuButton"
+              data-toggle="dropdown"
+              aria-haspopup="true"
+              aria-expanded="false"
+            >
+              <i class="fas fa-user"></i>
+              <i class="fas fa-caret-down"></i>
+            </button>
+            <div
+              class="dropdown-menu"
+              aria-labelledby="dropdownMenuButton"
+            >
+              <a
+                class="dropdown-item"
+                href="#"
+                @click="selectOption('Requested by Me', 'filter')"
+                ><i class="fas fa-user"></i> Requested by Me</a
```

Comment:
close the tag like this <i ... />
tag i must go in new line
close tag '>' in new line